api/proto/teleport/usertasks/v1/user_tasks_service.proto

```diff
@@ -32,7 +32,11 @@ service UserTaskService {
   // ListUserTasks returns a list of UserTasks. It supports pagination and filters.
   rpc ListUserTasks(ListUserTasksRequest) returns (ListUserTasksResponse);
   // ListUserTasksByIntegration returns a list of UserTasks filtered by an integration and other optional fields. It supports pagination.
-  rpc ListUserTasksByIntegration(ListUserTasksByIntegrationRequest) returns (ListUserTasksResponse);
+  // Deprecated: Use ListUserTasks instead.
+  rpc ListUserTasksByIntegration(ListUserTasksByIntegrationRequest) returns (ListUserTasksResponse) {
+    // TODO(marco): DELETE IN v19.0.0
+    option deprecated = true;
+  }
```

Comment:
As this is deprecated, should the implementation remain also?

Reply:
I replaced the implementation with `ListUserTasks` and pass in the integration as filter.
https://github.com/gravitational/teleport/blob/50504615651997c6488b327df1f080aa579dcb74/lib/auth/usertasks/usertasksv1/service.go#L210-L233

Do you think we should keep the old implementation? They should be compatible.

Reply:
Sorry I missed this implementation, ignore me